internal/savedjobs/model.go

```diff
@@ -0,0 +1,17 @@
+package savedJobs
+
+import (
+	"time"
+)
+
+type JobBookmark struct {
+	DeveloperID string
+	JobID       string
+	SavedAt     time.Time
+	AppliedAt   time.Time
+}
+
+type JobBookmarkRq struct {
+	DeveloperID string
+	JobID       int
```

Comment:
JobID is legacy and only used internally, for jobs we should be using externaljob id which is a char(27) ksuid token